crengine/src/private/lvfreetypeface.cpp

```diff
@@ -1035,32 +1041,41 @@ bool LVFreeTypeFace::getGlyphInfo(lUInt32 code, LVFont::glyph_info_t *glyph, lCh
             return fallback->getGlyphInfo(code, glyph, def_char, passMask);
         }
     }
-    int flags = FT_LOAD_DEFAULT;
-    flags |= (!_drawMonochrome ? FT_LOAD_TARGET_NORMAL : FT_LOAD_TARGET_MONO);
+    int rend_flags = FT_LOAD_DEFAULT;
+    rend_flags |= (!_drawMonochrome ? FT_LOAD_TARGET_NORMAL : FT_LOAD_TARGET_MONO);
```

Comment:
~~(This renaming feels a bit unnecessary :) For me, rend_flags is a bit more confusing as it's the variable name we use in lvrend.cpp for the enhanced rendering flags.)~~
I guess that's ok, we have that named rend_flags in some later part of the code :)